linkis-commons/linkis-rpc/src/main/scala/org/apache/linkis/rpc/conf/RPCConfiguration.scala

```diff
@@ -63,7 +63,7 @@ object RPCConfiguration {
 
   val PUBLIC_SERVICE_LIST: Array[String] = CommonVars(
     "wds.linkis.gateway.conf.publicservice.list",
-    "query,jobhistory,application,configuration,filesystem,udf,variable,microservice,errorcode,bml,datasource"
+    "cs,data-source-manager,metadataquery,query,jobhistory,application,configuration,filesystem,udf,variable,microservice,errorcode,bml,datasource"
```

Comment:
contextService is better

Reply:
the contextService's servicename is ps-cs,  this config is used to match the service name.

Reply:
old version  use linkis-ps-metadatamanager, to ensure compatibility, it is recommended to add this `metadatamanager`